lib/drivers/bayer/bayerContourNext.js

```diff
@@ -609,7 +765,7 @@ module.exports = function (config) {
     },
 
     processData: function (progress, data, cb) {
-      //debug('in processData');
+      debug('in processData');
```

Comment:
Should this be commented out (or even be here at all)?

Reply:
I'm not sure why it was commented out in the first place, but it makes sense for it to be there (as  it's also in the other drivers and helps to make sense of the driver logs).